app/graphql/types/query_type.rb

```diff
@@ -36,7 +36,8 @@ def farmer_search(name:)
     end
 
     def farmer_by_id(id:)
-      Farmer.where(id: id)
+      # Farmer.where(id: id)
+      Farmer.find(id)
```

Comment:
I'm glad we were able to figure this out. Saves us from unnecessarily iterating over the result when we know we're only going to get one back.